changelog/_unreleased/2023-08-31-fix-export-tempfile-closed-before-copying-data.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Fix export temporary file closed before copying data
+issue:
+author: Cuong Huynh
+author_email: cuong.huynh@pluszwei.io
+author_github: cuonghuynh
+---
+# Administration
+* Fixed the export to work with Google Bucket adapter
```

Comment:
The right keyword is "Changed", and please mention where you changed `\Shopware\Core\Content\ImportExport\Processing\Writer\AbstractFileWriter::flush`